File: Pratica 6 - Funcoes 2/java_logic/Exercicio1_v2.java
import java.util.Scanner;

public class Exercicio1_v2 {
    static void verificar_numero(int num) {
        if (num % 2 == 0) {
            System.out.println("Par");
        } else {
            System.out.println("Ímpar");
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Digite um número: ");
        int num = sc.nextInt();
        verificar_numero(num);
        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio1_v2.cpp
#include <iostream>
using namespace std;

void verificarPar(int num) {
    if (num % 2 == 0)
        cout << "Par";
    else
        cout << "Ímpar";
}

int main() {
    int n;
    cout << "Digite um número inteiro: ";
    cin >> n;
    verificarPar(n);
    return 0;
}